### DISCUSSION TITLE
notes/gis/GIS%E5%9F%BA%E7%A1%80/%E4%BB%80%E4%B9%88%E6%98%AFGIS

### DISCUSSION THREAD

**Discussion by giscus** [dllcnx/dllcnx.github.io#46] [Announcements]
# notes/gis/GIS%E5%9F%BA%E7%A1%80/%E4%BB%80%E4%B9%88%E6%98%AFGIS

什么是地理信息系统 地理信息系统 (GIS) 是由计算机硬件，软件和不同的方法组成的系统。来支持空间数据的采集、存储、管理、处理、分析、建模以及显示，以便解决复杂的规划和管理问题。通过关联看似不相关的数据，GIS 可以帮助个人和组织更好地理解空间模式和关系。 1 gis应用 GIS技术是空间数据基础设施的重要组成部分。 GIS可以使用包括位置在内的任何...

https://dllcnx.com:10000/notes/gis/GIS%E5%9F%BA%E7%A1%80/%E4%BB%80%E4%B9%88%E6%98%AFGIS.html

**Comment by dllcnx** [comment]
345345
